pkg/master/ports/ports.go

```diff
@@ -26,12 +26,12 @@ const (
 	// SchedulerPort is the default port for the scheduler status server.
 	// May be overridden by a flag at startup.
 	SchedulerPort = 10251
-	// ControllerManagerPort is the default port for the controller manager status server.
+	// InsecureKubeControllerManagerPort is the default port for the controller manager status server.
 	// May be overridden by a flag at startup.
-	ControllerManagerPort = 10252
```

Comment:
Given that out of tree providers may be linking against this, its probably worth noting in the release notes.

Reply:
> Given that out of tree providers may be linking against this, its probably worth noting in the release notes.

We should kill these constants.  Experience in vendoring now says that no one in their right mind should bring in kube for a constant like this.

Reply:
> Given that out of tree providers may be linking against this, its probably worth noting in the release notes.

We usually do not document dev-only changes, which reminds me that we really want `dev-release-note` blocks in the descriptions and collect them for a dev changelog on release.